starter-code/vue.js

```diff
@@ -0,0 +1,120 @@
+Vue.component('message', {
```

Comment:
Components should be named with PascalCase. Also, you want them to contain at least two words.
Henceforth, the name would be `BulmaMessage`.